spec/replication/evm_dbsync_spec.rb

```diff
@@ -48,6 +48,15 @@ class ::MasterDb < ActiveRecord::Base; end
     #   based on the configuration
   end
 
+  example ":replicate_backlog" do
```

Comment:
What is `example`?  Is it just another way to specify an `it`? If so, can this just be `it` for consistency?